src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -157,34 +201,22 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
         // correction via Edit UI
         this.setCurrentFiling(filing)
         this.setFileCorrectionDialog(true)
-        break
+        return
 
       case FilingTypes.CHANGE_OF_ADDRESS:
       case FilingTypes.CHANGE_OF_DIRECTORS:
         if (this.isBaseCompany || this.isCoop) {
           // correction via Edit UI if current type is BEN/BC/CC/ULC/C/CBEN/CCC/CUL or CP
           this.setCurrentFiling(filing)
           this.setFileCorrectionDialog(true)
-          break
-        } else {
-          // Local correction otherwise
-          this.$router.push({
-            name: Routes.CORRECTION,
-            params: { correctedFilingId: filing.filingId.toString() }
-          })
-          break
+          return
         }
-
-      case FilingTypes.ANNUAL_REPORT:
-      case FilingTypes.CONVERSION:
-      default:
-        // local correction for all other filings
-        this.$router.push({
-          name: Routes.CORRECTION,
-          params: { correctedFilingId: filing.filingId.toString() }
-        })
```

Comment:
No more local corrections.

However, I'll leave the route and view in place in case we ever want them back. Thoughts?

Reply:
Makes sense. Will be useful in future if needed.